Lib/fontbakery/profiles/fontwerk.py

```diff
@@ -1,17 +1,22 @@
 """
 Checks for Fontwerk <https://fontwerk.com/>
 """
+import re
 
 from fontbakery.callable import check
 from fontbakery.section import Section
 from fontbakery.status import PASS, FAIL
 from fontbakery.fonts_profile import profile_factory
 from fontbakery.message import Message
 from fontbakery.profiles.googlefonts import GOOGLEFONTS_PROFILE_CHECKS
+from fontbakery.profiles.fontwerk_conditions import general
```

Comment:
Not sure if we should rather use `profile.configuration_defaults` for this, it would make the checks more portable, but it also has other implications.

I personally would put all that data directly into this profile, but maybe there are other opinions about this around. @felipesanches?